package.json

```diff
@@ -12,13 +12,15 @@
     "preversion": "npm run test:lint && npm run build && npm test && git commit --allow-empty -am \"Update lib\""
   },
   "main": "lib/index.js",
+  "typings": "index.d.ts",
```

Comment:
Would it make sense to port the whole code to TypeScript and drop the type definition file?

Reply:
We could still generate JS + typedefs based on that.

Reply:
I'm sorry, i'm not using this library often, I'm just wanted to learn ts

Reply:
No worries. 👍